recipes/reckless/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
```

```suggestion
conan_basic_setup(TARGETS)

find_package(reckless CONFIG TARGET)

add_executable(${PROJECT_NAME} test_package.cpp)
target_link_libraries(${PROJECT_NAME} reckless::reckless)
target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
```

https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#cmake-targets

---

Ok, it will be hard to keep track of official and unofficial targets in CCI for maintainers, but I already gave my opinion on that topic.